pkg/validation/validate_cluster.go

```diff
@@ -364,6 +364,18 @@ func (v *ValidationCluster) validateNodes(cloudGroups map[string]*cloudinstances
 				}
 
 				v.Nodes = append(v.Nodes, n)
+			} else if n.Role == "apiserver" {
+				if !ready {
+					v.addError(&ValidationError{
+						Kind:          "Apiserver",
```

Comment:
```suggestion
						Kind:          "APIServer",
```

Reply:
Unfortunately, `kops rolling-update cluster --instance-group-role <role>` title-case the input. so the consistent user-facing name need to be `Apiserver`